### ISSUE TITLE
setZoomLevel doesn't zoom 20%

### ISSUE THREAD

**Issue by fancyydk** [electron/electron#11869]
* Electron version: 1.7.12
* Operating system: Windows and macOS

### Expected behavior

https://electronjs.org/docs/all#contentssetzoomlevellevel says the following:
![image](https://user-images.githubusercontent.com/12807659/36011350-55addd40-0d0b-11e8-99f7-d73d2f044b65.png)

I expect setZoomLevel to increase/decrease zoom 20% with each level increased/decreased in the following manner:

Level -3: 40%
Level -2: 60%
Level -1: 80%
**Level 0: 100% (default)**
Level 1: 120%
Level 2: 140%
Level 3: 160%

(% increase/decrease with respect to the original level 0)

### Actual behavior

Level -3: 57.87%
Level -2: 69.4%
Level -1: 83.3%
Level 0: 100%
Level 1: 120%
Level 2: 144%
Level 3: 172.8%

It looks as though zoom increase adds 20% to the immediately preceding zoom level. For example
Level 1: 120% = 20% increase from 100%
Level 2: 144% = 20% increase from 120%, not 40% increase from 100%
Level 3: 172.8% = 20% increase from 144%, not 60% increase from 100%

If this was the case, shouldn't I get 140% when the initial zoom level is 0 and setZoomLevel(2) is called? It's still 144%.

While these are 20% increases in a different way than I expected, zoom decreases by a different %.
Level -1: 83.3% = 16.67% decrease from 100%
Level -2: 69.4% = 16.67% decrease from 83.3%
Level -3: 57.87% = 16.67% decrease from 69.4%

I don't even know why 16.67% instead of 20% for decrease.

setZoomFactor, on the other hand, works as I described in the Expected behavior section.

### How to reproduce

  $ git clone https://github.com/fancyydk/electron-quick-start.git -b dokang/bug/zoom-level-factor
  $ npm install
  $ npm start || electron .

You can play around with this sample app to test different zoom levels and zoom factors.

### Zoom level change

**Default**
![image](https://user-images.githubusercontent.com/12807659/36011134-3b6f753e-0d0a-11e8-917d-db8069d7ba87.png)

**Zoom level +1**
![image](https://user-images.githubusercontent.com/12807659/36011181-84af2b2c-0d0a-11e8-89c0-33f23888b68c.png)

**Zoom level +2**
![image](https://user-images.githubusercontent.com/12807659/36011161-68603e52-0d0a-11e8-81f1-70f673806a56.png)

**Zoom level +3**
![image](https://user-images.githubusercontent.com/12807659/36011167-73c8c1d8-0d0a-11e8-98fd-570ab4eb8cac.png)

**Zoom level -1**
![image](https://user-images.githubusercontent.com/12807659/36011185-93fc799a-0d0a-11e8-899f-6110be55892a.png)

**Zoom level -2**
![image](https://user-images.githubusercontent.com/12807659/36011192-9e597488-0d0a-11e8-8cb0-09392043a2f3.png)

**Zoom level -3**
![image](https://user-images.githubusercontent.com/12807659/36011204-a8764162-0d0a-11e8-9f5f-bea32ca8188d.png)

### Zoom factor change

**Default**
![image](https://user-images.githubusercontent.com/12807659/36011134-3b6f753e-0d0a-11e8-917d-db8069d7ba87.png)

**Zoom factor +0.2**
![image](https://user-images.githubusercontent.com/12807659/36011270-e6008d30-0d0a-11e8-825f-b7e30f30a978.png)

**Zoom factor +0.4**
![image](https://user-images.githubusercontent.com/12807659/36011278-f0869614-0d0a-11e8-86b9-1100146441ed.png)

**Zoom factor +0.6**
![image](https://user-images.githubusercontent.com/12807659/36011283-fa508466-0d0a-11e8-91b5-bb086daa82fc.png)

**Zoom factor -0.2**
![image](https://user-images.githubusercontent.com/12807659/36011306-14bb60c8-0d0b-11e8-9057-7fcfb512d905.png)

**Zoom factor -0.4**
![image](https://user-images.githubusercontent.com/12807659/36011309-1d850132-0d0b-11e8-8bf9-aa63c0214a76.png)

**Zoom factor -0.6**
![image](https://user-images.githubusercontent.com/12807659/36011323-2c233e16-0d0b-11e8-9574-64e93ca87253.png)

**Comment by MarshallOfSound** [comment]
@fancyydk The "20%" in the docs is actual an exponential 20%, I.e. 20% of the previous level added again.  The exact formula is

```
scale = 1.2 ^ zoomLevel
```

So your 144% is gained through `1.2 ^ 2` 😄

**Comment by codebytere** [comment]
Like @MarshallOfSound said it's actually expected behavior; we'll make this more clear in the docs :)

**Comment by fancyydk** [comment]
@MarshallOfSound @codebytere 
That I kind of figured :) That still didn't make sense for negative zoom levels because even after understanding the positive scales, I kept thinking negative scales would be something like 1 - 0.2 ^ abs(negative zoom level) and I was way off because zoom level -1 was not 0.8 ( = 1 - 0.2 ^ 1 ). And then I realized negative zoom levels are just -1 = 1.2 ^ -1 = 83.33%, -2 = 1.2 ^ -2 = 69.4% and so on.

Thanks for the clarification. I think the doc can/should be more clear about this.
